src/blocks/plugins/sticky/index.js

```diff
@@ -48,9 +51,9 @@ const StickyExtension = ( el, props ) => {
 					label={ __( 'Transform to Sticky', 'otter-blocks' ) }
```

Comment:
We can exclude this from the workflow. Right now, we enable Transform to Stick when you enable the tool, so it doesn't make sense to include this as default. No one wants to make Sticky as default. Plus, if you disable it then the Sticky settings are disabled too.